### ISSUE TITLE
xft/bmf: Lost text in menu's and tab bars between commits

### ISSUE THREAD

**Issue by buhl** [raboof/notion#58]
On 32784ef8c7e95ccc38af1c36a414b037793e59f7 things looked fine
![32784ef8c7e95ccc38af1c36a414b037793e59f7](https://user-images.githubusercontent.com/221842/31634363-83b0bfac-b2c3-11e7-9d49-bd291adbf72b.png)

But with the following commit (313f4086d06345d3f8d5af0a5afd24ee5b852d5b) I lost text.
![313f4086d06345d3f8d5af0a5afd24ee5b852d5b](https://user-images.githubusercontent.com/221842/31634362-8399af2e-b2c3-11e7-9565-6af286cb2269.png)

When I diff the two I get
```diff
--- 32784ef8/system-autodetect.mk	2017-10-16 23:07:04.200584545 +0200
+++ 313f4086/system-autodetect.mk	2017-10-16 23:03:55.117604136 +0200
@@ -112,6 +112,9 @@
     X11_INCLUDES += `pkg-config xft --cflags`
     X11_LIBS += `pkg-config xft --libs`
     DEFINES += -DHAVE_X11_XFT
+    DEFINES += -DHAVE_X11_BMF
+else
+    DEFINES += -DHAVE_X11_BMF
 endif
```

And I can see from the build log that both flags are enabled
`x86_64-pc-linux-gnu-gcc [...]  -DHAVE_X11_XFT -DHAVE_X11_BMF [...]`

I can supply more info if needed. just let me know.

**Comment by raboof** [comment]
That's weird, @ideasman42 any idea?

**Comment by ideasman42** [comment]
@buhl could you upload your notion config, and interested in output from the command `xdpyinfo | grep -B2 resolution`

**Comment by knixeur** [comment]
This was probably fixed by 55b7fbf8e563bebc040e963f33742b20cbed6dd1

**Comment by raboof** [comment]
I suspect so as well. Closing, feel free to reopen when the issue is still there.
